src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs

```diff
@@ -2512,7 +2512,7 @@ private DataObject Copy(NormalizedSnapshotSpanCollection spans)
             {
                 var text = GetText(spans);
                 var blocks = GetTextBlocks(spans);
-                var rtf = _rtfBuilderService.GenerateRtf(spans, TextView);
+                var rtf = ((IRtfBuilderService2)_rtfBuilderService).GenerateRtf(spans, CancellationToken.None);
```

Comment:
You could also cast/store it as an IRtfBuilderService2 up front to avoid this bit of awkwardness here (if desired).